scrapers/TACAmateurs.yml

```diff
@@ -0,0 +1,26 @@
+name: TACAmateurs
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - tacamateurs.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //div[@id="update_wrapper"]/h1
+     Date:
+        selector: //div[@class="date_wrapper"]
+        postProcess:
+          - parseDate: 02-January-2006
+     Performers:
+      Name: //div[@id="facepic_wrapper"]/h2
+     Details: //div[@id="update_wrapper"]/p[1]
```

Comment:
depending on the performer there are scenes with more paragraphs
eg `https://www.tacamateurs.com/sites/Hot-Milf/my-new-bustier-pt2/24490/`
I did a limited testing with 
```yaml
     Details: 
       selector: //div[@id="update_wrapper"]/p[not(@class) and not(@align)]
       concat: "\n\n"
```
and it seemed ok. It might need a little more testing to make sure no extra p are added.

Reply:
Yeah that works ok, Gonna get one past ya one day that does not need editing hahahahahahaha

Reply:
You already had a few that didnt need any editing :-)
Usualy if the site is structured ok there is nothing to notice. I blame the site devs for not keeping it simple and tidy :P

Reply:
Yeah there is normally at least one  thing that is  a pain in the arse to try and scrape.